### DISCUSSION TITLE
ASP.NET Core API Boilerplate HealthCheck Endpoint

### DISCUSSION THREAD

**Discussion by mikepc** [Dotnet-Boxed/Templates#173] [Q&A]
The docs for the API boilerplate say that the HealthCheck functionality is enabled by default, but I can't find the actual URL/endpoint to call it with. I could use some guidance on how to go about configuring this

**Comment by mikepc** [comment]
Note that after starting up the template I tried the following URLs that returned 404:

/healthCheck
/HealthCheck
/healthcheck
/status
/api/status
/Status
/api/Status

I'm thinking that it really isn't enabled by default, but in that case the docs need to be updated with information on enabling it.

**Comment by m3talstorm** [comment]
Also experiencing this, did you ever find the endpoint?

**Comment by VictorioBerra** [comment]
To check if it's enabled see startup.cs and verify its been registered.  If you don't see it there you can easily add it. 

https://github.com/Dotnet-Boxed/Templates/blob/master/Source/ApiTemplate/Source/ApiTemplate/Startup.cs#L141

**Comment by m3talstorm** [comment]
I meant the URL, I'm blind, its `/status`
